R/format_output.R

```diff
@@ -0,0 +1,85 @@
+
+
+#' Collapse age groups in output
+#'
+#' Sums over age groups for each compartment
+#'
+#' @param d output data.frame
+#'
+#' @return Output data.frame
+collapse_age <- function(d){
+  d %>%
+    dplyr::group_by(.data$compartment, .data$t, .data$replicate) %>%
+    dplyr::summarise(y =  sum(.data$y)) %>%
+    dplyr::ungroup()
+}
+
+#' Collapse compartments in output
+#'
+#' Sums over simplified groups across compartments
+#'
+#' @param d output data.frame
+#'
+#' @return Output data.frame
+collapse_compartment <- function(d){
+  d %>%
+    dplyr::mutate(group = dplyr::case_when(
+      grepl("IMVGet", .data$compartment) ~ "ICU",
```

Comment:
think we want to report demand on ICU rather than ICU beds filled. So maybe `grepl("IMV", .data$compartment) ~ "ICU"` which should then get the IMVNotGet compartments. 

Same as well for IOx